package.json

```diff
@@ -54,7 +54,6 @@
     "keycloak-auth-utils": "^3.3.0",
     "log": "^1.4.0",
     "mathjs": "^5.10.3",
-    "minimist": "^1.2.8",
```

Comment:
@Prajwal17Tunerlabs this is a deletion. which means this package was already in use can you verify where it is used.